spec/models/request_spec.rb

```diff
@@ -285,63 +285,19 @@
-      end
-
-      describe 'only sends to contributors of the organization' do
-        it { should change { Message.count }.from(0).to(2) }
-        it { should change { Message.pluck(:recipient_id).sort }.from([]).to([1, 2]) }
-        it { should change { Message.pluck(:sender_id) }.from([]).to([request.user.id, request.user.id]) }
-        it { should change { Message.pluck(:broadcasted) }.from([]).to([true, true]) }
-        it { should change { Message::File.count }.from(0).to(2) }
-      end
-
-      describe 'given a planned request' do
-        before { request.schedule_send_for = 1.hour.from_now }
-
-        let!(:admin) { create_list(:user, 2, admin: true) }
-        let!(:other_organization) { create(:organization, users_count: 2) }
-
-        it_behaves_like 'an ActivityNotification', 'RequestScheduled', 3
-      end
-    end
+    describe 'given a planned request' do
+      before { request.schedule_send_for = 1.hour.from_now }
 
-    describe 'creates message only for contributors tagged with tag_list' do
-      let(:request) do
-        Request.new(
-          title: 'Hitchhiker’s Guide',
-          text: 'What is the answer to life, the universe, and everything?',
-          tag_list: 'programmer',
-          user: user,
-          organization: organization
-        )
-      end
-      before(:each) do
-        create(:contributor, id: 1, email: 'somebody@example.org', tag_list: ['programmer'], organization: organization)
-        create(:contributor, id: 2, email: nil, telegram_id: 22, organization: organization)
-      end
+      let!(:admin) { create_list(:user, 2, admin: true) }
+      let!(:other_organization) { create(:organization, users_count: 2) }
 
-      it { should change { Message.count }.from(0).to(1) }
-      it { should change { Message.pluck(:recipient_id) }.from([]).to([1]) }
-      it { should change { Message.pluck(:sender_id) }.from([]).to([request.user.id]) }
-      it { should change { Message.pluck(:broadcasted) }.from([]).to([true]) }
+      it_behaves_like 'an ActivityNotification', 'RequestScheduled', 3
     end
 
-    describe 'given contributors who are deactivated' do
-      before(:each) do
-        create(:contributor, :inactive, id: 3, email: 'deactivated@example.org', organization: organization)
-        create(:contributor, id: 4, email: 'activated@example.org', organization: organization)
-        create(:contributor, :inactive, id: 5, telegram_id: 24, organization: organization)
+    it 'schedules a job to broadcast the request' do
+      expect { subject.call }.to have_enqueued_job(BroadcastRequestJob).with do |params|
+        expect(params[:request_id]).to eq(request.id)
       end
-
-      it { should change { Message.count }.from(0).to(1) }
-      it { should change { Message.pluck(:recipient_id) }.from([]).to([4]) }
-      it { should change { Message.pluck(:sender_id) }.from([]).to([request.user.id]) }
-      it { should change { Message.pluck(:broadcasted) }.from([]).to([true]) }
```

Comment:
These specs have been migrated to the `spec/jobs/broadcast_request_job_spec.rb` See b5db6640